src/modules/price-router/PriceRouter.sol

```diff
@@ -3,28 +3,48 @@ pragma solidity 0.8.16;
 
 import { ERC20, SafeTransferLib } from "src/base/ERC4626.sol";
 import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
-import { AutomationCompatibleInterface } from "@chainlink/contracts/src/v0.8/interfaces/AutomationCompatibleInterface.sol";
 import { IChainlinkAggregator } from "src/interfaces/external/IChainlinkAggregator.sol";
 import { SafeCast } from "@openzeppelin/contracts/utils/math/SafeCast.sol";
 import { Math } from "src/utils/Math.sol";
 import { Address } from "@openzeppelin/contracts/utils/Address.sol";
-import { ICurvePool } from "src/interfaces/external/ICurvePool.sol";
-import { IAaveToken } from "src/interfaces/external/IAaveToken.sol";
+import { Extension } from "src/modules/price-router/Extensions/Extension.sol";
+import { Registry } from "src/Registry.sol";
+
+import { UniswapV3Pool } from "src/interfaces/external/UniswapV3Pool.sol";
+import { OracleLibrary } from "@uniswapV3P/libraries/OracleLibrary.sol";
 
 /**
  * @title Sommelier Price Router
  * @notice Provides a universal interface allowing Sommelier contracts to retrieve secure pricing
  *         data from Chainlink.
- * @author crispymangoes, Brian Le
+ * @author crispymangoes
  */
-contract PriceRouter is Ownable, AutomationCompatibleInterface {
+contract PriceRouter is Ownable {
```

Comment:
A general question for the `PriceRouter` design:

What is the current design's response to oracles failing? Let's say the stETH : USD oracle goes down. Obviously, a lot of bad things will happen to the rest of DeFi, but it would be ideal for the Sommelier setup to isolate and mitigate contagion. 

Do we currently just have everything revert related to rebalances? I guess the assets themselves are still in the cellar possession, and rebalances just won't work. I'm basically wondering what our stance is on fallback and contingent oracle sources since there is some risk vector when working with just one oracle source (Chainlink) for example for an asset price. There's pro's and con's to fallback design vs not, and so I maybe we should open this as an issue to discuss.

wydt?

Reply:
Yes we currently have everything revert. Which is the safest thing for us. It is kinda rough for lending protocols that want to use our Cellar shares, but that is why setting up balancer 80/20 pools would be nice for them, since they can emergency liquidate people using the pool.

Reply:
We have discussed this, sounds good!